openapi/connector_mgmt-private-admin.yaml

```diff
@@ -543,6 +543,19 @@ paths:
     delete:
       tags:
         - Connector Clusters Admin
+      parameters:
+        - name: namespace_id
+          description: The id of the namespace to delete
+          schema:
+            type: string
+          in: path
+          required: true
+        - name: force
+          description: Flag to force deletion of namespace in Fleet manager if true
+          schema:
+            type: string
```

Comment:
should this be a boolean?

Reply:
Json schema boolean only supports true and false apparently. I made it the same as connector force delete API and used golang boolean parse to support setting it to `1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False`.

Reply:
I see, part of the api use boolean though and it seems more standard and plain in its intent to be.

Reply:
Fixed boolean param processing for all connector admin endpoints.